src/core/entities/EmployerModel.js

```diff
@@ -0,0 +1,38 @@
+import xlsx from 'xlsx-populate';
+
+export default class EmployerModel {
+    constructor(
+        id,
+        fio,
+        tabelNumber,
+        sex,
+        birthdayDate,
+        disabilityGroup,
+        startDate,
+        title,
+        titleCode,
+        workPlace,
+    ) {
+        this.id = id;
+        this.fio = fio;
+        this.tabelNumber = tabelNumber;
+        this.sex = sex;
+        this.birthdayDate = birthdayDate;
+        this.disabilityGroup = disabilityGroup;
+        this.startDate = startDate;
+        this.title = title;
+        this.titleCode = titleCode;
+        this.workPlace = workPlace;
+    }
+
+    convert() {
+        this.startDate = xlsx.numberToDate(this.startDate);
```

Comment:
is it some kind of type conversion function ?

Reply:
Yes, It converts numbers to JS date format. Because xlsx file has another date format